src/Test/Unit/WarmUp/UrlsPattern/CategoryCmsPageTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Test\Unit\WarmUp\UrlsPattern;
+
+use Codeception\PHPUnit\TestCase;
+use Magento\MagentoCloud\App\GenericException;
+use Magento\MagentoCloud\WarmUp\UrlsPattern\CategoryCmsPage;
+use Magento\MagentoCloud\WarmUp\UrlsPattern\CommandArgumentBuilder;
+use Magento\MagentoCloud\WarmUp\UrlsPattern\ConfigShowUrlCommand;
+use PHPUnit\Framework\MockObject\MockObject;
+
+/**
+ * @inheritDoc
+ */
+class CategoryCmsPageTest extends TestCase
+{
+    /**
+     * @var CategoryCmsPage
+     */
+    private $categoryCmsPage;
+
+    /**
+     * @var ConfigShowUrlCommand|MockObject
+     */
+    private $configShowUrlCommandMock;
+
+    /**
+     * @var CommandArgumentBuilder|MockObject
+     */
+    private $argumentBuilderMock;
+
+    /**
+     * @inheritDoc
+     */
+    protected function setUp()
+    {
+        $this->configShowUrlCommandMock = $this->createMock(ConfigShowUrlCommand::class);
+        $this->argumentBuilderMock = $this->createMock(CommandArgumentBuilder::class);
+
+        $this->categoryCmsPage = new CategoryCmsPage($this->configShowUrlCommandMock, $this->argumentBuilderMock);
+    }
+
+    /**
+     * @param string $pattern
+     * @param array $expectedUrls
+     * @dataProvider getUrlsDataProvider
+     * @throws GenericException
+     */
+    public function testGetUrls(string $pattern, array $expectedUrls)
+    {
+        $urls = [
+            'http://example.com/',
+            'http://example2.com/',
+            'http://site.com/',
+            'http://test.com/'
+        ];
+
+        $this->argumentBuilderMock->expects($this->once())
+            ->method('generate')
+            ->with('cms-page', '*')
+            ->willReturn(['--entity=cms-page']);
+        $this->configShowUrlCommandMock->expects($this->once())
+            ->method('execute')
+            ->with(['--entity=cms-page'])
+            ->willReturn($urls);
+
+        $this->assertEquals(
+            $expectedUrls,
+            array_values($this->categoryCmsPage->getUrls('cms-page', $pattern, '*'))
+        );
+    }
+
+    /**
+     * @return array
+     */
+    public function getUrlsDataProvider(): array
+    {
+        return [
+            [
+                '*',
+                [
+                    'http://example.com/',
+                    'http://example2.com/',
+                    'http://site.com/',
+                    'http://test.com/'
+                ],
+            ],
+            [
+                '/example/',
+                [
+                    'http://example.com/',
```

Comment:
how it is expected URLs if you should take url path, without domain?
e.g. customer has website URL http://caterina.com and he wants to warm-up cms page http://caterina.com/cat/, when he also has dozen other cms pages (about, contacts, category, ...). What pages will be warm-upped in the result?
Please add such cases

Reply:
Tests updated.

Reply:
yes, but I meant more, what if path is the same as part of domain **cat**erina.com with path **cat**. Seems like in this case all cms page URL will be warm-upped

Reply:
Yes, but I can't change this as it was already released.

Reply:
you can. I guess it is a bug. not sure that someone expect this

Reply:
Ok, fixed.

Reply:
in the description of this method (in class) you wrote that pattern may be `"/path/to/page" - exact page match` . I would mention that you should not cover `path` with slash.
To show this in test, please add to the list of URL 4 more URLs ( and remove 'http://example1.com/cat' and 'http://example2.com/cat'):
``` 
- http://example.com/example/
- http://example.com/example1/
- http://example.com/example/path
- http://example.com/path/example/path
```
and add next patterns/paths
```
example
/example/
/example/path/
```
and case with `'/http.*/'` is redundant

Reply:
Also, need to add these difference in docs (about slash in path) 
@meker12  FYI ^^

Reply:
Done